addons/project/static/src/components/project_right_side_panel/components/project_milestone.xml

```diff
@@ -5,7 +5,7 @@
         <div class="list-group mb-2">
             <div class="o_rightpanel_milestone list-group-item list-group-item-action d-flex justify-content-evenly px-0 cursor-pointer" t-att-class="state.colorClass">
                 <span t-on-click="toggleIsReached">
-                    <i class="fa position-absolute pt-1" t-att-class="state.checkboxIcon"/>
+                    <i class="fa position-absolute pt-1" t-att-class="state.checkboxIcon" t-attf-title="{{ !milestone.is_reached ? 'Mark as reached' : 'Mark as incomplete'}}"/>
```

Comment:
This will not be translatable because the title is dynamic. I believe you should do something like this:
```suggestion
                    <t t-if="milestone.is_reached" t-set="title">Mark as incomplete</t>
                    <t t-else="" t-set="title">Mark as reached</t>
                    <i class="fa position-absolute pt-1" t-att-class="state.checkboxIcon" t-attf-title="title"/>
```
To check if it works, otherwise you would need to implement a getter on the component and translate the text there

Reply:
Thank you for the suggesion! I could make it work using `t-att-title` but not `t-attf-title`, I wonder if that changes anything